src/WpfMath/Controls/FormulaControl.xaml.cs

```diff
@@ -125,7 +127,7 @@ private void Render()
             var visual = new DrawingVisual();
             var renderer = texFormula.GetRenderer(TexStyle.Display, Scale, SystemTextFontName);
 
-            var selectionBrush = SelectionBrush;
+            var selectionBrush = (SolidColorBrush)SelectionBrush;
```

Comment:
This is right? By the way, I did not find this for Avalonia

Reply:
Probably better
```c#
var objColor = SelectionBrush
            .GetType()
            .GetField(nameof(Color))
            .GetValue(null);
if (objColor is Color selectionColor)
```

Reply:
But then, it’s better to set `Color` instead of a `Brush` immediately in the control properties

Reply:
> This is right? By the way, I did not find this for Avalonia

fix to pattern matching, because it was a bug

Reply:
It is better to replace `Brush` with our `ArgbColor` abstraction if we don't support any non-solid color brushes anymore.

Reply:
The difference will be that the user himself will have to create _knowing_ colors. When, as in wpf / avalonia, there is already a set of colors